api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -475,6 +475,21 @@
      * Only supports string codecs that have a constructor with no arguments
      */
     Attribute<Map<Class<?>, Class<? extends StringCodec<?>>>> STRING_CODECS = new Attribute<Map<Class<?>, Class<? extends StringCodec<?>>>>(new Map2String<Class<?>, Class<? extends StringCodec<?>>>(",", "=", new Class2String<Object>(), new Class2String<StringCodec<?>>()));
+
+    /**
+     * The number of consecutive container failures that should lead to
+     * blacklisting of nodes by application master
+     */
+    Attribute<Integer> MAX_CONSECUTIVE_CONTAINER_FAILURES_FOR_BLCKLIST = new Attribute<Integer>(Integer.MAX_VALUE);
```

Comment:
Should also mention that default value means it is off, which is the current behavior.